docs/user-guide/docker-volume-cli.md

```diff
@@ -1,93 +1,103 @@
 ```
-Docker volumes are backed by VMDKs. VMDKs are attached to the VM in which Docker is requesting for a volume during Docker run. VMDKs can be attached in [different modes.](http://cormachogan.com/2013/04/16/what-are-dependent-independent-disks-persistent-and-non-persisent-modes/)
-
-Currently the following are supported
-
-1. [persistent](http://cormachogan.com/2013/04/16/what-are-dependent-independent-disks-persistent-and-non-persisent-modes/): If the VMDK is attached as persistent it will be part of a VM snapshot. If a VM snapshot has been taken while the Docker volume is attached to a VM, the Docker volume then continues to be attached to the VM that was snapshotted.
-2. [independent_persistent](http://cormachogan.com/2013/04/16/what-are-dependent-independent-disks-persistent-and-non-persisent-modes/): If the VMDK is attached as independent_persistent it will not be part of a VM snapshot. The Docker volume can be attached to any VM that can access the datastore independent of snapshots.
-
-### access (vSphere only)
-```
-docker volume create --driver=vsphere --name=MyVolume -o access=read-only -o diskformat=thin
-docker volume create --driver=vsphere --name=MyVolume -o access=read-write -o diskformat=thin (default)
-```
 
-The access mode determines if the volume is modifiable by containers in a VM. The access mode allows to first create a volume with write access and initialize it with binary images, libraries (for exmple), and subsequently change the access to "read-only" (via the admin CLI). Thereby, creating content sharable by all containers in a VM.
+#### Volume Cloning (clone-from)
 
-### fstype
-```
-docker volume create --driver=<vsphere/photon> --name=MyVolume -o size=10gb -o fstype=xfs
-docker volume create --driver=<vsphere/photon> --name=MyVolume -o size=10gb -o fstype=ext4 (default)
-```
+When creating a new volume, you can specificy a volume to clone and create a new one. This is a complete new volume of which you can change all parameters except size and fstype.
 
-Specifies which filesystem will be created on the new volume. vSphere Docker Volume Service will search for a existing /sbin/mkfs.**fstype** on the docker host to create the filesystem, and if not found it will return a list of filesystems for which it has found a corresponding mkfs. The specified filesystem must be supported by the running kernel and support labels (-L flag for mkfs). Defaults to ext4 if not specified. 
 
-### clone-from (vSphere only)
 ```
 docker volume create --driver=vsphere --name=CloneVolume -o clone-from=MyVolume -o access=read-only
 docker volume create --driver=vsphere --name=CloneVolume -o clone-from=MyVolume -o diskformat=thin (default)
 ```
 
-Specifies a volume to be cloned when creating a new volume. The created clone is completely independent from the original volume and will inherit the same options, which can be changed with the exception of the size and fstype.
- 
-### flavor (Photon only)
```

Comment:
As mentioned earlier, let's move all photon specific content to a separate section.